provision/vvv-init.sh

```diff
@@ -61,3 +61,22 @@ else
     sed -i "s#{{TLS_CERT}}##" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
     sed -i "s#{{TLS_KEY}}##" "${VVV_PATH_TO_SITE}/provision/vvv-nginx.conf"
 fi
+
+DELETE_DEFAULT_PLUGINS=`get_config_value 'delete_default_plugins' ''`
+if [ ! -z "${DELETE_DEFAULT_PLUGINS}" ]; then
+    noroot wp plugin delete akismet
+    noroot wp plugin delete hello
+fi
+
+WP_PLUGINS=`get_config_value 'plugins' ''`
+if [ ! -z "${WP_PLUGINS}" ]; then
+    for plugin in ${WP_PLUGINS//- /$'\n'}; do 
+        noroot wp plugin install "${plugin}" --activate
```

Comment:
We also want to document that all plugins get activated on install too

Reply:
if this pr will be merged I will update this https://github.com/Varying-Vagrant-Vagrants/custom-site-template/pull/16

Reply:
I'd rather we document it here so it's a self contained PR